### PR TITLE
Callback Logs Cleanup Pt. 2

### PR DESCRIPTION
**Pull request by sampocs**
Closes: #XXX

## Context and purpose of the change
Continuing the callback logs cleanup with the remaining ICACallbacks.

## Brief Changelog
* Cleaned up logs for the claim, reinvest, and rebalance callbacks
* Separated `LogCallbackWithHostZone` into `LogICACallbackWithHostZone` and `LogICQCallbackWithHostZone`
* Campfire rule - misc cleanup/standardization across functions (e.g. variable naming)

## Author's Checklist

I have...

- [X] Run and PASSED locally all GAIA integration tests
- [ ] If the change is contentful, I either:
    - [ ] Added a new unit test OR 
    - [ ] Added test cases to existing unit tests
- [ ] OR this change is a trivial rework / code cleanup without any test coverage

If skipped any of the tests above, explain.

## Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] manually tested (if applicable)
- [ ] confirmed the author wrote unit tests for new logic
- [ ] reviewed documentation exists and is accurate

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes? 
  - [ ] Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`?
  - [ ] This pull request updates existing proto field values (and require a backend and frontend migration)? 
  - [ ] Does this pull request change existing proto field names (and require a frontend migration)?
  How is the feature or change documented? 
      - [ ] not applicable
      - [ ] jira ticket `XXX` 
      - [ ] specification (`x/<module>/spec/`) 
      - [ ] README.md 
      - [ ] not documented

### CODE REVIEW

File: utils/utils.go [hunk 1]
```diff
@@ -212,20 +212,27 @@ func LogWithHostZone(chainId string, s string, a ...any) string {
 	return fmt.Sprintf("|   %-13s |  %s", strings.ToUpper(chainId), msg)
 }
 
+// Returns a log string with a chain Id and callback as a prefix
+// callbackType is either ICACALLBACK or ICQCALLBACK
+// Format:
+//   |   CHAIN-ID    |  {CALLBACK_ID} {CALLBACK_TYPE}  |  string
+func logCallbackWithHostZone(chainId string, callbackId string, callbackType string, s string, a ...any) string {
+	msg := fmt.Sprintf(s, a...)
+	return fmt.Sprintf("|   %-13s |  %s %s  |  %s", strings.ToUpper(chainId), strings.ToUpper(callbackId), callbackType, msg)
```
Comment (riley-stride) [inline utils/utils.go:221]: Can you explain the trimming of the last 13 chars here?

File: x/stakeibc/keeper/icacallbacks_reinvest.go [hunk 2]
```diff
@@ -33,50 +36,61 @@ func (k Keeper) UnmarshalReinvestCallbackArgs(ctx sdk.Context, reinvestCallback
...
 	record := recordstypes.DepositRecord{
-		Amount:             amount,
-		Denom:              denom,
+		Amount:             reinvestCallback.ReinvestAmount.Amount,
+		Denom:              reinvestCallback.ReinvestAmount.Denom,
 		HostZoneId:         reinvestCallback.HostZoneId,
 		Status:             recordstypes.DepositRecord_DELEGATION_QUEUE,
 		Source:             recordstypes.DepositRecord_WITHDRAWAL_ICA,
-		DepositEpochNumber: epochNumber,
+		DepositEpochNumber: currentStrideEpochNumber,
```
Comment (riley-stride) [inline x/stakeibc/keeper/icacallbacks_reinvest.go:91]: why are we referencing attributes directly in 86-87, yet creating a temp var in 82 to be used here?

**Review by riley-stride — APPROVED** [review]
Looks good to me. Campfire rule is going to start paying some big dividends soon!

File: x/stakeibc/keeper/icacallbacks_claim_test.go [hunk 3]
```diff
@@ -39,7 +39,7 @@ func (s *KeeperTestSuite) SetupClaimCallback() ClaimCallbackTestCase {
 		// after a user calls ClaimUndelegatedTokens, the record is set to claimIsPending = true
 		// to prevent double claims
 		ClaimIsPending: true,
-		Amount: sdk.ZeroInt(),
+		Amount:         sdk.ZeroInt(),
```
[the remark this answers is not in the document]
Comment (sampocs) [reply x/stakeibc/keeper/icacallbacks_claim_test.go:42]: I think that's just my VSCode settings? I assumed it was the same as everyone else. 

I think being vertically aligned is expected (see the structs at the top of this file). It's possible that this line was changed with a different linter

File: x/stakeibc/keeper/icacallbacks_reinvest.go (hunk 2 above)
Comment (sampocs) [reply x/stakeibc/keeper/icacallbacks_reinvest.go:91]: Good eye - that wasn't intentional! I can remove the `currentStrideEpochNumber` temp variable. 
But I do think in that scenario it provides a little extra clarification over using just the attribute (i.e. that we're creating an DR with the _current_ epoch).

I could go either way though! Wdyt?

File: x/stakeibc/keeper/icacallbacks_claim_test.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (riley-stride) [reply x/stakeibc/keeper/icacallbacks_claim_test.go:42]: Can you check on this? CI lint check is failing

File: x/stakeibc/keeper/icacallbacks_reinvest.go (hunk 2 above)
Comment (riley-stride) [reply x/stakeibc/keeper/icacallbacks_reinvest.go:91]: I'd err on the side of consistency, not a big deal either way though so will fully defer to you

Comment (sampocs) [reply x/stakeibc/keeper/icacallbacks_reinvest.go:91]: updated!

File: x/stakeibc/keeper/icacallbacks_claim_test.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (sampocs) [reply x/stakeibc/keeper/icacallbacks_claim_test.go:42]: let me know though if you were referring to something else

**Pull request merged**